### PR TITLE
Update ug

### PR DESCRIPTION
**Pull request by papataco14**
Fixed typos and suggestions based on issues thread.

Related to #294

### CODE REVIEW

**Review by tayruxin — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -167,8 +168,8 @@ A command line has 4 components, **command word**, **index**, **prefix** and **p
 | Parameter    | `John Doe` | Represents the updated value or content. <br/> These values, which are provided by the user, replaces the `UPPER_CASE` placeholders in the command format |
```
Comment (GlendaChong) [inline docs/UserGuide.md:168]: I thought u wanted to edit the description?

File: docs/images/quick-start/githubRelease.png
Comment (GlendaChong) [inline docs/images/quick-start/githubRelease.png:1]: image isn't of our latest jar file though, does it matter that the MB size might be different?

**Review by GlendaChong — CHANGES_REQUESTED** [review]

File: docs/images/quick-start/githubRelease.png
Comment (tayruxin) [reply docs/images/quick-start/githubRelease.png:1]: I think can update after we do the final release

File: docs/UserGuide.md (hunk 1 above)
Comment (papataco14) [reply docs/UserGuide.md:168]: image

i was actually referring to the `John Doe` part. shorten to John?

File: docs/images/quick-start/githubRelease.png
Comment (papataco14) [reply docs/images/quick-start/githubRelease.png:1]: The contributions section will look the same. MB size I don't think matters.

**Review by GlendaChong — APPROVED** [review]

**Pull request merged**